lib/bootsnap/load_path_cache/cache.rb

```diff
@@ -152,7 +152,7 @@ def push_paths_locked(*paths)
 
       def unshift_paths_locked(*paths)
         @store.transaction do
-          paths.map(&:to_s).reverse.each do |path|
+          paths.map(&:to_s).reverse_each do |path|
             p = Path.new(path)
```

Comment:
This is totally unrelated but I think it might not be a good idea to use `p` as a variable name. `p` is a method defined in the Kernel module https://ruby-doc.org/core-2.4.1/Kernel.html#method-i-p

Reply:
I don't have much of a problem with it; it's clear from context that it's not being used as a function, nor would we be pretty-printing things to stdout from here.